### PR TITLE
New plugin - adjacent rooms

### PR DESCRIPTION
**Pull request by jon-heard**

### CODE REVIEW

**Comment by jon-heard** [comment]
Fixing ESLint now (forgot to check pre-PR).

**Comment by jon-heard** [comment]
Ok.  ESLint is good now.

File: src/adjacent rooms.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		// Turn each cell of text into either a number or null
+		return cells.map(item => {
+			return parseInt(item, 10) || null;
+		});
+	});
+	// Turn the cell data into adjacency lists
+	for (let y = 0; y < roomMap.length; y++) {
+		for (let x = 0; x < roomMap[y].length; x++) {
+			const roomId = roomMap[y][x];
+			// NOTE: map id's start at 1 so falsy checks are fine.
```
Comment (seleb) [inline src/adjacent rooms.js]: although bipsi starts map IDs at 1 in the editor, 0 is still a supported map ID, and older games/templates may still use it as an ID. i happen to be on an older PC atm and this caused some issues while i was testing, but seems simple to fix so i'll push an update shortly

Comment (jon-heard) [reply src/adjacent rooms.js]: Thanks for catching this.  i'll keep 0 ids as valid for different object types going forward.

**Pull request merged**